plugins/woocommerce/client/blocks/tests/e2e/tests/product-collection/inspector-controls.block_theme.spec.ts

```diff
@@ -553,10 +553,12 @@ test.describe( 'Product Collection: Inspector Controls', () => {
 
 			await expect( pageObject.products ).toHaveCount( 18 );
 
-			await page.getByLabel( 'Toggle block inserter' ).click();
-			await page.getByRole( 'tab', { name: 'Patterns' } ).click();
+			const addBlockButton = editor.canvas
+				.locator( '[data-type="woocommerce/product-collection"]' )
+				.getByRole( 'button', { name: 'Add block' } );
+			await addBlockButton.click();
 			await page.getByPlaceholder( 'Search' ).fill( 'product filters' );
-			await page.getByLabel( 'Product Filters' ).click();
+			await page.getByText( 'Product Filters' ).click();
```

Comment:
Not a big deal, but I think this can be replaced with Gutenberg utils, which I would expect to be a bit more performant and less brittle to break in future updates:

```TS
			const productCollectionBlock = await editor.getBlockByName(
				'woocommerce/product-collection'
			);
			const productCollectionClientId =
				( await productCollectionBlock
					.last()
					.getAttribute( 'data-block' ) ) ?? '';
			await editor.insertBlock(
				{ name: 'woocommerce/product-filters' },
				{ clientId: productCollectionClientId }
			);
```

Reply:
Good idea, thanks for the suggestion. I pushed up changes.